codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerFrameDeflateDecoderTest.java

```diff
@@ -60,19 +59,18 @@ public void testCompressedFrame() {
         // test
         assertNotNull(uncompressedFrame);
         assertNotNull(uncompressedFrame.content());
-        assertTrue(uncompressedFrame instanceof BinaryWebSocketFrame);
```

Comment:
why was this removed ?

Reply:
Because we read above https://github.com/netty/netty/blob/721d2c02da639bcc61cd1659921591018a4c7f3d/codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerFrameDeflateDecoderTest.java#L57